tests/PHPUnit/Framework/TestCase/IntegrationTestCase.php

```diff
@@ -92,6 +93,7 @@ public function setUp(): void
 
         PiwikCache::getEagerCache()->flushAll();
         PiwikCache::getTransientCache()->flushAll();
+        EventDispatcher::getInstance()->clearCache();
```

Comment:
Can we put this in `Fixture::clearInMemoryCaches()`?

Reply:
I see we call that in tearDown only for some reason... seems like a mistake, and we should clear all of those caches here too...

Reply:
@diosmosis updated the PR

Reply:
@diosmosis seems clearing all caches at this point lets various integration tests start failing. Seems clearing all static instances is a problem at that point as some fixtures are manipulating those before.

Reply:
Reverted it so it only adds the additional cache clear, but left a note so it's clear why not all caches are cleared